### ISSUE TITLE
Title case in breadcrumb title not ideal in some cases

### ISSUE THREAD

**Issue by matthisamoto** [youandmedigital/craft-breadcrumb#6]
As the plugin uses the slug to construct the breadcrumb title, I end up with strange title case words when I have abbreviations or other preferred non-capitalized text, e.g. the title "ABC News" would read "Abc News" in the breadcrumb title.

I adjusted this locally for myself by replacing this line in [BreadcrumbService.php](https://github.com/youandmedigital/craft-breadcrumb/blob/master/src/services/BreadcrumbService.php#L69):
```
$title = ucwords(str_replace(array('-', '_'), ' ', $segment));
```

with this to grab the entry title rather than use the slug:
```
// get entry for segment
$entry = Entry::find()
    ->slug($segment)
    ->one();
$title = $entry['title'];
```

I don't know if something like this would be preferred as a setting, like your custom field handle setting works for the current entry, so that a user could choose between the entry title or the slug-constructed title, so I didn't work up a PR.

**Comment by jonleverrier** [comment]
Hi @matthisamoto 

Thanks for taking time to report this issue and provide feedback.

I agree with you, in some scenarios the title case formatting is not ideal. 

In hindsight, I should of left the formatting of crumbs up to you via CSS text-transform. Although this approach would not fix situations like `ABC News` turning into `Abc News`.

Be careful with your changes - if you have a breadcrumb on a category element it will break. It will also break if you're using any custom routing (non-elements).

I'm going to have a think and try fix this issue.

**Comment by jonleverrier** [comment]
Just to let you know @matthisamoto, i've changed how Breadcrumb generates crumb titles which should fix your problem.

You're welcome to try if out on the development branch:

https://github.com/youandmedigital/craft-breadcrumb/blob/development/CHANGELOG.md

https://github.com/youandmedigital/craft-breadcrumb/blob/development/src/services/BreadcrumbService.php

**Comment by jonleverrier** [comment]
Just released [v1.1.0](https://github.com/youandmedigital/craft-breadcrumb/releases/tag/v1.1.0)

**Comment by matthisamoto** [comment]
@jonleverrier You rock, thank you for looking at this so quickly! I'm going to test this out tomorrow.

edit: totally works, thanks!
